kvmagent/kvmagent/plugins/virtiofs_plugin.py

```
@@ -292,30 +300,30 @@ def mount_virtiofs_in_vm(domain, tag, mount_path):
-        # 2. Check if already mounted
+        # 2. Check if already mounted (use shorter timeout)
         check_mount_cmd = "mountpoint -q %s" % safe_mount_path
         try:
-            exitcode, _, _ = qga.guest_exec_bash(check_mount_cmd)
+            exitcode, _, _ = qga.guest_exec_bash(check_mount_cmd, retry=QGA_MKDIR_TIMEOUT)
             if exitcode == 0:
                 logger.info("Path[%s] is already a mount point in VM[%s]" % (mount_path, domain.name()))
                 return True, None
         except Exception:
             # mountpoint returns non-zero if not a mount point, which raises exception
             pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)